### DISCUSSION TITLE
Add support for deserializing Layer.stretch

### DISCUSSION THREAD

**Discussion by zlovatt** [KraftyFX/aex#64] [Ideas]
This is technically possible, but using it starts messing with the timing / duration of the layer, keyframes, in/out points, etc etc.

Doable but requires work.

https://github.com/KraftyFX/aex/blob/147ea7e2bf13d1a8b993100ac516cb0629954458/estk/src/layer/layer.tsx#L112

**Comment by rafikhan** [comment]
We should probably mention this in the documentation and leave a voodoo comment in the codebase that links to this issue.
